### PR TITLE
Defined endpoints for file compression and added missing imports in sender.py

### LINKED ISSUE #791: Endpoints for file compression and missing imports
**Describe the bug**
* File compression endpoints are not defined.
* Some imports are missing in `sender.py` 

**To Reproduce**
1. Click the download button for any file from the completed queue
2. You find that the feature breaks

**Expected behavior**
* On clicking the download file should be download 
* Share button should also be working

### PR DESCRIPTION
**Pull request by kmehant**
## Description

* Defined endpoints for file compression
* Added missing imports in `sender.py`

## Related Issue
Fixes #791 

## Motivation and Context

This change is required for existing file compression and file download feature to work.

## How Has This Been Tested?

This has been tested locally using Docker containers

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kmehant** [comment]
@vivonk @agentmilindu Please review this [PR](#792 )

File: components/core/routes/Download.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def kill():
 			p.join()
 			jsonreq = json.dumps({'jsonrpc': '2.0', 'id': 'qwer', 'method': 'aria2.pauseAll'})
 			jsonreq = jsonreq.encode('ascii')
-			c = urllib.request.urlopen('http://localhost:6800/jsonrpc', jsonreq)
+			# c = urllib.request.urlopen('http://localhost:6800/jsonrpc', jsonreq)
```
Comment (vivonk) [inline components/core/routes/Download.py]: You can remove the unnecessary code instead of commenting it out :)

Comment (kmehant) [reply components/core/routes/Download.py]: Sure :)

**Comment by kmehant** [comment]
@vivonk @agentmilindu Shall we merge this to develop?

**Pull request merged**